packages/flutter_tools/lib/src/windows/build_windows.dart

```diff
@@ -165,23 +166,42 @@ Future<void> buildWindows(WindowsProject windowsProject, BuildInfo buildInfo, {
   }
 }
 
+String getCmakeWindowsArch(TargetPlatform targetPlatform)
+{
+  switch (targetPlatform) {
+    case TargetPlatform.windows_x64:
+      return 'x64';
+    case TargetPlatform.windows_arm64:
+      return 'ARM64';
+    case TargetPlatform.android:
+    case TargetPlatform.ios:
+    case TargetPlatform.darwin:
+    case TargetPlatform.linux_x64:
+    case TargetPlatform.linux_arm64:
+    case TargetPlatform.fuchsia_arm64:
+    case TargetPlatform.fuchsia_x64:
+    case TargetPlatform.tester:
+    case TargetPlatform.web_javascript:
+    case TargetPlatform.android_arm:
+    case TargetPlatform.android_arm64:
+    case TargetPlatform.android_x64:
+    case TargetPlatform.android_x86:
+      throwToolExit('Unsupported target platform "$targetPlatform".');
+  }
+}
```

Comment:
Please make this function private and put `{` on same line as function declaration.

Also, I'd omit the unsupported platforms to reduce verbosity, but feel free to keep them if you prefer being explicit.

```suggestion
String _getCmakeWindowsArch(TargetPlatform targetPlatform) {
  return switch (targetPlatform) {
    TargetPlatform.windows_x64 => 'x64',
    TargetPlatform.windows_arm64 => 'ARM64',
    _ => throwToolExit('Unsupported target platform "$targetPlatform".'),
  };
}
```

Reply:
Changed with your version.